### DISCUSSION TITLE
macOS 15

### DISCUSSION THREAD

**Discussion by PashCode** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1329] [Q&A] [answered]
Hey, guys. Will the app work properly on the new macOS version or wait for updates with your comments?

**Comment by MatthiasKerbl** [comment, marked as answer]
> Hey, guys. Will the app work properly on the new macOS version or wait for updates with your comments?

Hi @PashCode,

We have been optimizing AlDente for macOS 15 for weeks, and it generally works. However, some minor issues arose after the final release and we will release a new update soon.

Best,
Matthias
